sw/device/tests/meson.build

```diff
@@ -132,10 +136,25 @@ foreach sw_test_name, sw_test_lib : sw_tests
       build_by_default: true,
     )
 
+    sw_test_dv_frames = []
+    if device_name == 'sim_dv' and sw_test_name in dv_tests
```

Comment:
@sriyerg we can always change this if-statement, but this gives you the configurability you need, I hope.